docker/build-farmOS.sh

```diff
@@ -62,15 +54,5 @@ for plugin in ${allowedPlugins[@]}; do
   composer config --no-plugins allow-plugins.$plugin true
 done
 
-# Run composer install with optional arguments passed into this script.
-if [ $# -eq 0 ]; then
-  composer install
-else
-  composer install "$*"
-fi
-
-# Set the version in farm.info.yml.
-sed -i "s|version: 3.x|version: ${FARMOS_VERSION}|g" /var/farmOS/web/profiles/farm/farm.info.yml
-
-# Remove the Composer cache directory.
-rm -rf "$COMPOSER_HOME"
+# Create folder for composer installations.
+mkdir -p ${FARMOS_PATH}/web/sites/simpletest/browser_output
```

Comment:
This line has wrong description in comment. Also, it should be probably moved back to the dev docker image (maybe to the phpunit.sh script), build-farmOS.sh script is executed both for dev and prod images. For production, setup for executing automated tests is not needed.